graph/patterns/subsets.md

```diff
@@ -16,12 +16,18 @@ Existing patterns for this either have special-cased strings or have tightly cou
 
 Have an abstract base class where all variants of the subset are derived types from the base subset. For more information, see the [general subtyping guidance](./subtypes.md).
 
-The abstract base class should also hold an enum for all possible variants. The purpose of including this is to allow for easier ways to do query and filter operations on variants like `all` and `none` without relying on `isof` functions.
+The abstract base class may also optionally hold an `enum` for the different variants. If it does, the `enum` must have a member for all possible variants. The purpose of including this is to allow for easier ways to do query and filter operations on variants like `all` and `none` without relying on `isof` functions.
 
```

Comment:
Could you please include an example of a querie with  isof  and an example without?

Reply:
As much as I prefer examples from real SCDL  i think that it's better if CSDL examples match to query examples.